<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroup</class>
 <widget class="QWidget" name="AddGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*全局样式表*/
QWidget#AddGroup{
	background-color: rgb(255, 255, 255);
}

QLabel#groupImg{
	border:1px solid rgb(149, 149, 149);
}
</string>
  </property>
  <widget class="QLabel" name="groupImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../image/Image.qrc">:/new/Chat/icons/Chat/logo_Show.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>群昵称：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="group_Number">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>群号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="group_Name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
